src/v2/view-builder/views/ov/EnrollOVSwitchChannelView.js

```diff
@@ -0,0 +1,25 @@
+import { loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+import { isMobileDevice } from '../../../../util/BrowserFeatures';
+
+const Body = BaseForm.extend({
+  title () {
+    return isMobileDevice() ? loc('oie.enroll.okta_verify.setup.title', 'login'):
+      loc('oie.enroll.okta_verify.select.channel.title', 'login');
+  },
+  getUISchema () {
+    const schemas = BaseForm.prototype.getUISchema.apply(this, arguments);
+    const description = {
```

Comment:
I'd try to use `'label-top': true,` option and verify with UX.
You'd need to ask API send label for the field or manually add a label in `ion-object-handler` or here.

Reply:
Given it's so short, I thought it can be `label` hence asked and feels make more sense? (usually when we show radio buttons, we have label)

Reply:
I tired that but the spacing between input and explain is very little and adding it separately makes sense. Also technically it is not an explain text.